### PR TITLE
Dev ios

### PR DESCRIPTION
**Pull request by bklukaczewski**
https://refugeeinfo.atlassian.net/browse/RID-79
tested under both iOS and Android

### CODE REVIEW

File: src/scenes/ServiceDetails.js [hunk 1]
```diff
@@ -205,6 +121,9 @@ export default class ServiceDetails extends Component {
 
     getDirections(lat, long) {
         let location = `${lat},${long}`;
+        if (Platform.OS = 'ios'){
```
Comment (kkrampa) [inline src/scenes/ServiceDetails.js]: Should be ```===```

File: src/scenes/ServiceDetails.js [hunk 2]
```diff
@@ -205,6 +121,9 @@ export default class ServiceDetails extends Component {
...
+            Linking.openURL(`http://maps.apple.com/?daddr=${lat},${long}&dirflg=w&t=m`)
```
Comment (kkrampa) [inline src/scenes/ServiceDetails.js]: Shouldn't we put return statement here to avoid executing line 127?

File: index.android.js [hunk 3]
```diff
@@ -22,6 +22,8 @@ import { fetchRegionFromStorage } from './src/actions/region';
 
 store.dispatch(fetchRegionFromStorage());
 
+var styles = require('./src/styles');
```
Comment (kkrampa) [inline index.android.js]: Don't use var.

File: src/components/LocationListView.js [hunk 4]
```diff
@@ -87,17 +87,27 @@ export default class LocationListView extends Component {
 
     render() {
         const primary = this.props.primary;
-        if (this.props.rows.length === 0) {
+
+        if (this.props.rows.length == 0 && this.props.loaded) {
+            return (
+                <View style={styles.container}>
+                    <View style={styles.header}>
+                        <Text style={styles.headerText}>No locations found</Text>
```
Comment (kkrampa) [inline src/components/LocationListView.js]: I know that you don't added this but we should add this string to messages file.

File: src/components/LocationListView.js [hunk 5]
```diff
@@ -87,17 +87,27 @@ export default class LocationListView extends Component {
...
             return (
-                <View style={styles.noLocationsContainer}>
-                    <Text
-                        style={styles.noLocationsText}
-                    >
-                            There are no locations
-                    </Text>
+                <View style={styles.container}>
+                    <View style={styles.header}>
+                        <Text style={styles.headerText}>Loading locations, please wait...</Text>
```
Comment (kkrampa) [inline src/components/LocationListView.js]: Same as in https://github.com/theirc/app.refugee.info/pull/93/files#diff-c937f2a56d3dfc79322d5c7f769d9b44R95

File: src/components/MapButton.js [hunk 6]
```diff
@@ -1,6 +1,8 @@
 import React, { Component, PropTypes } from 'react';
 import { StyleSheet, TouchableHighlight, Image } from 'react-native';
 
+var styles = require('../styles');
```
Comment (kkrampa) [inline src/components/MapButton.js]: Don't use var

File: src/styles.js [hunk 7]
```diff
@@ -0,0 +1,180 @@
+'use strict';
+
+var React = require('react-native');
```
Comment (kkrampa) [inline src/styles.js]: It's better to use ES6 import styles here.

File: src/styles.js [hunk 8]
```diff
@@ -0,0 +1,180 @@
...
+
+import { Platform } from 'react-native';
+
+var styles = React.StyleSheet.create({
```
Comment (kkrampa) [inline src/styles.js]: Don't use var

File: src/styles.js [hunk 9]
```diff
@@ -0,0 +1,180 @@
...
+        marginTop: 5
+    },
+    validationText: {
+        color: '#a94442',
+        fontSize: 12,
+        marginTop: -5
+    }
+});
+
+module.exports = styles;
```
Comment (kkrampa) [inline src/styles.js]: Use ES6 export instead of commonjs.

File: src/scenes/initial.js [hunk 10]
```diff
@@ -0,0 +1,64 @@
+import React, {Component, PropTypes} from 'react';
+import {AsyncStorage, Image, StyleSheet, View} from 'react-native';
+import {connect} from 'react-redux';
+import DrawerCommons from '../utils/DrawerCommons';
+
+var styles = require('../styles');
```
Comment (kkrampa) [inline src/scenes/initial.js]: Don't use var.

File: src/scenes/LanguageSelection.js [hunk 11]
```diff
@@ -1,13 +1,14 @@
...
 import Spinner from 'react-native-loading-spinner-overlay';
-import { connect } from 'react-redux';
-import { Drawer, Button } from 'react-native-material-design';
+import {connect} from 'react-redux';
+import {Drawer, Button} from 'react-native-material-design';
 
 import DrawerCommons from '../utils/DrawerCommons';
 import I18n from '../constants/Messages';
 
+var styles = require('../styles');
```
Comment (kkrampa) [inline src/scenes/LanguageSelection.js]: Don't use var.

File: index.android.js (hunk 3 above)
Comment (kkrampa) [inline index.android.js]: Also, you can import styles using es6 import in all places where styles are imported.

**Comment by kkrampa** [comment]
Mostly minor comments. Only one blocker: https://github.com/theirc/app.refugee.info/pull/93/files?diff=split#r66910827

**Comment by bklukaczewski** [comment]
thanks @kkrampa ! changes done and pushed

**Comment by kkrampa** [comment]
:+1:

**Pull request merged**